monkey-runner/sample/buildSrc/build.gradle

```diff
@@ -0,0 +1,19 @@
+apply plugin: 'groovy'
+
+repositories {
+    jcenter()
+}
+
+// these must mirror the ones in the `plugin` module
+// the dependencies are not transitive (we specify only sourceset below)
+dependencies {
+    compile gradleApi()
+    compile 'com.novoda:gradle-android-command-plugin:1.6.2'
+}
```

Comment:
I avoided doing this so the dependencies closure was visible in each file - I tried to extract just the contents of the block but it didn't like it.

The other change (extracting the dependencies to a single file) should address half the beef with this comment, at least (don't have keep the gradle android command plugin version synced in two places).